styles.css

```diff
@@ -163,6 +171,19 @@ button {
     padding: 10px;
 }
 
+#savePaletteButton {
```

Comment:
Button styling should match the rest of the website and be targeted with classes.

Reply:
Just noticed the styling for `#savePaletteButton` is identical to `#deletePaletteButton` on line 169. We should target them with a class.